split-gpg2-client

```diff
@@ -1,21 +1,35 @@
-#!/bin/bash
+#!/bin/bash --
+set -euo pipefail
```

Comment:
I used to default to `-e` too, but changed my mind. It's just too easy to fool yourself with it. `-o pipefail` applied. If you insist I'm ok with `-u`, but it didn't safe me so far and seems likely to run into the false positives.